elements/font.go

```diff
@@ -1,8 +1,59 @@
 package elements
 
+import "math"
+
 type FontInfo struct {
 	Name        string
-	Width       int
-	Height      int
+	Width       float64
+	Height      float64
 	Orientation FieldOrientation
 }
+
+func (font FontInfo) GetSize() float64 {
+	switch font.Orientation {
+	case FieldOrientation90, FieldOrientation270:
+		return font.Width
+	default:
+		return font.Height
+	}
+}
+
+var bitmapFontSizes = map[string][2]float64{
+	"A": {9, 5},
+	"B": {11, 7},
+	"C": {18, 10},
+	"D": {18, 10},
+	"E": {28, 15},
+	"F": {26, 13},
+	"G": {60, 40},
+	"H": {21, 13},
```

Comment:
What is this list exactly? Why does it have more entries than you have the font files?

Reply:
Those are ZPL bitmap fonts and their sizes, not our fonts
Basically you can't freely scale bitmap fonts (everything that is not font 0) and their width and height should always be font's base width and height * some int multiplier
![Screenshot from 2024-03-27 09-04-20](https://github.com/ingridhq/zebrash/assets/78364711/4f04b30d-7d48-46b8-ab99-f62994460c2c)